libvirt/tests/src/virsh_cmd/pool/virsh_pool_autostart.py

```
@@ -87,7 +87,8 @@ def check_pool(pool_name, pool_type, checkpoint,
                     error_msg = "Dir pool should be always active when libvirtd restart. "
                     error_msg += "See https://bugzilla.redhat.com/show_bug.cgi?id=1238610"
                     logging.error(error_msg)
-                test.fail("Pool %s isn't %s as expected" % (checkpoint, expect_value))
+                else:
+                    test.fail("Pool %s isn't %s as expected" % (checkpoint, expect_value))
```

if "checkpoint == 'State' and pool_type in ("dir", "scsi")" is NOT an error, why defining error_msg info in it?

---

Ignored this error before, and now have already changed error to debug info.